API.yaml

```diff
@@ -1893,7 +1893,7 @@ components:
       description: Represents the style on an element (ex. Entry, TimeGraphState, ...) returned by any output. 
         Supports style inheritance. To avoid having too many styles, the element style can have a parent style and will have all the same style property values as the parent and can add or override style properties.
       properties:
-        parentStyleKey:
+        parentKey:
```

Comment:
If you change it to parentKey, then the Trace Compass server needs to be updated (add a serializer)

Reply:
I think, we'll have to do this in all cases. We shouldn't make the the protocol match whatever names we choose in Trace Compass, but rather have meaningful simple speak-for-themselves names in the protocol and have Trace Compass comply through serializers. Anyway, if we change the hasRowModel above to hasData, we'll have to do this too for other objects.

Reply:
I agree, the protocol names need to be meaningful and should be independent of the Trace Compass internal naming.